### PR TITLE
Authentication provider changes

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/auth-provider-setup/byo-jwt-provider/jsonwebtoken.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  { algorithm: "ECDSA", keyid: "1bb9605c36e69386830202b2d" }, // <-- Replace it with your kid. This has to be present in the JWKS endpoint.
+);
+
+console.log(token);
+```
+
+</TabItem>
+</Tabs>
+
+## Create JWKS
```
Comment (shahbaz17) [inline docs/auth-provider-setup/byo-jwt-provider/jsonwebtoken.mdx:136]: Please make this modular. as it's being used in all three, would help us in maintaining in future.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/auth-provider-setup/byo-jwt-provider/jsonwebtoken.mdx (hunk 1 above)
Comment (AyushBherwani1998) [reply docs/auth-provider-setup/byo-jwt-provider/jsonwebtoken.mdx:136]: Ah, yes. Makes sense.

File: docs/auth-provider-setup/byo-jwt-provider/jose.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Create JWT using jose package
+sidebar_label: JWT using jose
```
Comment (yashovardhan) [inline docs/auth-provider-setup/byo-jwt-provider/jose.mdx:3]: This sidebar label looks repeated. Just having jose, jsonwebtoken, passport-jwt etc will look better.

**Review by yashovardhan — COMMENTED** [review]
Overall looks good. There are missing docs like Telegram login, but I think @ihsraham might be working on it.

@shahbaz17 will check and merge

File: docs/auth-provider-setup/byo-jwt-provider/jose.mdx (hunk 2 above)
Comment (AyushBherwani1998) [reply docs/auth-provider-setup/byo-jwt-provider/jose.mdx]: Done

**Review by shahbaz17 — APPROVED** [review]

**Pull request merged**